css/MobileStyle.css

```diff
@@ -437,4 +437,154 @@ body {
   border-radius: 8px;
 }
 
-/* About section start */
+/* About section end */
+
+/* Contact form start */
+
+#form-section {
+  width: 100%;
+  height: auto;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+#form-section .form-wrapper {
+  width: 375px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  background-image: url("./images/Frame.png");
```

Comment:
- [ ] [OPTIONAL] Please fix the path so your image can be shown properly. Add an extra `.` at the beginning.  😎️